.clang-format

```diff
@@ -9,7 +9,7 @@ AllowShortLoopsOnASingleLine: false
 AllowShortFunctionsOnASingleLine: false
 AllowShortIfStatementsOnASingleLine: false
 AlwaysBreakTemplateDeclarations: false
-ConstructorInitializerAllOnOneLineOrOnePerLine: false
+PackConstructorInitializers: NextLine
```

Comment:
why did you change this?

Reply:
I asked @theyueli to do this. Apparently, clang formatter deprecated `ConstructorInitializerAllOnOneLineOrOnePerLine` option which leads to weird formatting on newer formatters.